### ISSUE TITLE
[Android] onPress of individual parts of a row don't trigger

### ISSUE THREAD

**Issue by PhilippKrone** [650Industries/react-native-invertible-scroll-view#16]
Hi,

I'm using the invertible scroll view to render various chat messages. Some chat messages contain clickable text. I'm using Touchable*s to make parts of the text clickable, which is working fine in iOS. In Android, it seems that the Listview is infront of the individual rows (is this possible?!) and thus the onPress events are not even fired. Using the inspector, whenever I try to click on an individual row, the entire listview is selected and not the row.

Any idea what the reason might be?

Thanks!

**Comment by PhilippKrone** [comment]
P.S. if it helps, my code of the list view is as follows:

``` javascript
        <ListView
          renderScrollComponent={props => <InvertibleScrollView {...props} inverted />}
          dataSource={this.state.dataSource}
          renderRow={this._renderAttributeRow}
          onEndReached={this.onEndReached}
          onEndReachedThreshold={50}
          scrollsToTop={false}
        />
```

**Comment by ide** [comment]
Hmm... one thing that could be happening is that Android might not transform its hit boxes when you transform a view. That is, when we invert the ScrollView, it's hit box doesn't get inverted and so touch events don't go to the places you are touching.

I'm not sure how to fix it on Android but if it's something React Native can address, that'd be the best outcome. Otherwise `transform` is much less useful on Android.

**Comment by PhilippKrone** [comment]
Mhh, do you have an idea how to debug that thing? I've tried yesterday but I dont really know where to look

**Comment by ide** [comment]
Not sure off the top of my head. I'd grep around for View and Touch code in Android and set a couple of breakpoints where it seems like touches are handled.

**Comment by PhilippKrone** [comment]
Indeed it seems like the hit box is not inverted. To press an item at the  bottom of the scrollview requires me to press an empty area at the very top. :(

Update: I have just opened https://github.com/facebook/react-native/issues/3812

**Comment by PhilippKrone** [comment]
@ide Do you see a chance to work around this RN issue? This thing is kinda blocking for me and I'd be happy for any work around possible? :/

**Comment by ide** [comment]
I don't know how to work around this since the touch events aren't firing, and we need to use a transform.

**Comment by ide** [comment]
This should be fixed in the upcoming 0.17 / master. 
https://github.com/facebook/react-native/issues/3557
